client/src/main/java/org/dst/client/DstStringProxy.java

```diff
@@ -42,4 +43,22 @@ public String get(String key) throws DstException {
 
     return response.getValue();
   }
+
+  public void del(String key){
```

Comment:
It seem that the method is not needed in `StringProxy`, it should be `drop` I think.